tests/components/mill/test_config_flow.py

```diff
@@ -4,14 +4,15 @@
 
 from homeassistant import config_entries
 from homeassistant.components.mill.const import CLOUD, CONNECTION_TYPE, DOMAIN, LOCAL
+from homeassistant.components.recorder import Recorder
 from homeassistant.const import CONF_IP_ADDRESS, CONF_PASSWORD, CONF_USERNAME
 from homeassistant.core import HomeAssistant
 from homeassistant.data_entry_flow import FlowResultType
 
 from tests.common import MockConfigEntry
 
 
-async def test_show_config_form(hass: HomeAssistant) -> None:
+async def test_show_config_form(recorder_mock: Recorder, hass: HomeAssistant) -> None:
```

Comment:
On most of these config flow tests, mock setup so it's not loading the rest of the integrations (less resource intensive that mocking the recorder since we anyhow isn't testing it)
https://github.com/home-assistant/core/blob/dev/tests/components/airtouch5/test_config_flow.py#L12